tasks/Task.cpp

```diff
@@ -143,93 +153,44 @@ void* Task::startTaskFunc(void* argument)
         return 0;
     }
 
-    // Prepare the LibManager and required configuration files
-    libManager = new lib_manager::LibManager();
-
-    std::string corelibsConfigPath;
-    // If the graphical interface should be disabled, the configuration 
-    // needs to exclude the shared library with graphics options
-    // Thus, the nogui configuration file needs to be used for the startup 
+    // todo: change this in a configuration of MARS
     if(marsArguments->enable_gui)
     {
-        corelibsConfigPath = marsArguments->config_dir + "/core_libs.txt";
+        std::string cmd;
+        cmd = "cp ";
+        cmd += marsArguments->config_dir + "/core_libs.txt.example ";
```

Comment:
I don't know in detail either.
But as @malter  said all the .txt are not needed in general anymore.
Don't know how the behaviour for the orogen/mars integration is currently.
@malter: Are the *libs.txt needed for orogen/mars currently?